[2024-06-04T18:37:33.901+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-04T18:37:33.983+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_ETL_directions.transform_load_directions_data manual__2024-06-04T18:36:27.490203+00:00 [queued]>
[2024-06-04T18:37:34.018+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_ETL_directions.transform_load_directions_data manual__2024-06-04T18:36:27.490203+00:00 [queued]>
[2024-06-04T18:37:34.022+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-06-04T18:37:34.094+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_load_directions_data> on 2024-06-04 18:36:27.490203+00:00
[2024-06-04T18:37:34.140+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=596) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-04T18:37:34.151+0000] {standard_task_runner.py:63} INFO - Started process 598 to run task
[2024-06-04T18:37:34.155+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_ETL_directions', 'transform_load_directions_data', 'manual__2024-06-04T18:36:27.490203+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/dag_ETL_directions.py', '--cfg-path', '/tmp/tmpm0_i4b2h']
[2024-06-04T18:37:34.164+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask transform_load_directions_data
[2024-06-04T18:37:34.427+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_ETL_directions.transform_load_directions_data manual__2024-06-04T18:36:27.490203+00:00 [running]> on host 6b740a15ab9e
[2024-06-04T18:37:34.795+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matheus' AIRFLOW_CTX_DAG_ID='dag_ETL_directions' AIRFLOW_CTX_TASK_ID='transform_load_directions_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T18:36:27.490203+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T18:36:27.490203+00:00'
[2024-06-04T18:37:34.799+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-04T18:37:34.933+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-06-04T18:37:35.525+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-04T18:37:35.535+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-04T18:37:35.565+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_ETL_directions, task_id=transform_load_directions_data, run_id=manual__2024-06-04T18:36:27.490203+00:00, execution_date=20240604T183627, start_date=20240604T183733, end_date=20240604T183735
[2024-06-04T18:37:35.645+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-04T18:37:35.726+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-04T18:37:35.739+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
